frame/frame-layer.go

```diff
@@ -60,6 +60,19 @@ func NewFrameLayer(transportLayer io.ReadWriter) *Layer {
 	go frameLayer.bgWork()
 	go frameLayer.bgRead()
 
+	// Empty the buffer
+empty:
+	for {
+		select {
+		case <-parserInput:
```

Comment:
Instead of relying on go frameLayer.bgRead() to empty the buffer, could we perhaps consume what is on transportLayer instead?

Reply:
See #10